src/manual-multiple/web/index.js

```diff
@@ -4,6 +4,23 @@ import { cleanupButtons } from '../../_shared/js/creatives';
 cleanupButtons();
 reportClicks();
 
+// logo javascript
+          getIframeId()
```

Comment:
There's a bunch of code duplication here. Let's chat this through in person 👍

Reply:
All fixed and re-factored (thank you!).